tools/circle_plus_gen/main.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import typing
+
+from lib.circle_plus import CirclePlus
+from lib.train_param import TrainParam
+
+
+def get_cmd_args():
+    parser = argparse.ArgumentParser(
+        prog='circle plus generator',
+        description='help handle circle file with training hyperparameters')
+
+    parser.add_argument(
+        'input_circle_file', metavar="input.circle", type=str, help='input circle file')
```

Comment:
(optional)

```suggestion
        'input', help='input circle file')
```

The default type of argument is string. Therefore, there is no need to specify the type as `str`. And I prefer the argument name to be simple to make the code concise.